script/utils/diamondEMERGENCYPauseGitHub.sh

```
@@ -0,0 +1,154 @@
+    sleep 3                    # wait for 3 seconds before trying the operation again
+  done
+
+  # check if call was executed successfully or used all attempts
+  if [ $ATTEMPTS -gt "$MAX_ATTEMPTS" ]; then
+    error "[network: $NETWORK] failed to pause diamond ($DIAMOND_ADDRESS)"
+    echo "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<< end network $NETWORK <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
+    return 1
+  fi
+
+  # try to call the diamond
+  OWNER=$(cast call "$DIAMOND_ADDRESS" "owner() external returns (address)" --rpc-url "$RPC_URL")
```

Shellcheck

[warning] OWNER appears unused. Verify use (or export if used externally).

(SC2034)